de.dlr.sc.virsat.external.lib.apache.poi/build.properties

```diff
@@ -1,8 +1,16 @@
 source.. = src/
 output.. = target/classes/
 bin.includes = META-INF/,\
-               .,\
                about.html,\
                about_files/,\
-               externalLib/
+               externalLib/,\
+               pom.xml,\
+               externalLib/maven/commons-codec.jar,\

```

Comment:
We probably don't need to explicitly list all jars

Reply:
Correct, we can try to just sue the folder once we update the versions